.github/workflows/good-first-issue.yml

```
@@ -15,8 +15,19 @@ jobs:
         with:
           issue-number: ${{ github.event.issue.number }}
           body: |
-            Please see [our Contributing guide](https://matplotlib.org/stable/devel/index) for
-            more information on how to contribute.
-            **We do not assign issues**, and unless there is a PR for this issue already, feel free
-            to start working on it. Do not hesitate to ask if something needs clarification - see
-            also [our different communication channels](https://matplotlib.org/stable/devel/contributing.html#get-connected).
+            *Good first issue - notes for new contributors*
```

```suggestion
            ### Good first issue - notes for new contributors
```
or alternatively bold. Italics is not prominent enough IMHO.